tests/tck/features/geo/GeoBase.feature

```diff
@@ -220,7 +220,7 @@ Feature: Geo base
     # Create index on geo column
     When executing query:
       """
-      CREATE TAG INDEX any_shape_geo_index ON any_shape(geo);
+      CREATE TAG INDEX any_shape_geo_index ON any_shape(geo) s2_max_level=30, s2_max_cells=8;
```

Comment:
prefer the following format:

```sql
CREATE TAG INDEX any_shape_geo_index ON any_shape(geo) WITH s2_max_level=30, s2_max_cells=8;
// OR 
CREATE TAG INDEX any_shape_geo_index ON any_shape(geo) WITH (s2_max_level=30, s2_max_cells=8);
```

with comment:

```sql
CREATE TAG INDEX any_shape_geo_index 
ON any_shape(geo) 
WITH s2_max_level=30, s2_max_cells=8 
COMMENT "geo index";
```

reference the postgresql: https://www.postgresql.org/docs/11/sql-createindex.html

Reply:
Any update?

Reply:
I will update this pr when I have time.

Reply:
> ```
> CREATE TAG INDEX any_shape_geo_index ON any_shape(geo) WITH (s2_max_level=30, s2_max_cells=8);
> ```
> 
> This format is also what I originally wanted!

Need to list `COMMENT` separately? So that the `IndexParams` just contains `s2_max_level` and `s2_max_cells`?

Reply:
I find the syntax : 
```
create_schema_prop_list
    : create_schema_prop_item {
        $$ = new SchemaPropList();
        $$->addOpt($1);
    }
    | create_schema_prop_list COMMA create_schema_prop_item {
        $$ = $1;
        $$->addOpt($3);
    }
    ;

create_schema_prop_item
    : KW_TTL_DURATION ASSIGN legal_integer {
        $$ = new SchemaPropItem(SchemaPropItem::TTL_DURATION, $3);
    }
    | KW_TTL_COL ASSIGN STRING {
        $$ = new SchemaPropItem(SchemaPropItem::TTL_COL, *$3);
        delete $3;
    }
    | comment_prop_assignment {
        $$ = new SchemaPropItem(SchemaPropItem::COMMENT, *$1);
        delete $1;
    }
    ;
```
There is no `WITH`, `(` or `)` @yixinglu

Reply:
It's different. I think max/min level is the former index property not to describe the statement.

Reply:
ok, got it.